service/tcp.go

```diff
@@ -265,6 +265,13 @@ func (s *tcpService) handleConnection(listenerPort int, clientTCPConn *net.TCPCo
 		logger.Debugf("proxy %s <-> %s", clientTCPConn.RemoteAddr().String(), tgtConn.RemoteAddr().String())
 		ssw := ss.NewShadowsocksWriter(clientConn, cipherEntry.Cipher)
 		ssw.SetSaltGenerator(cipherEntry.SaltGenerator)
+		mss, err := onet.GetMSS(clientConn)
```

Comment:
You can call this on `clientTCPConn` directly

Reply:
Good idea! Done